google-signin-aware.html

```diff
@@ -460,13 +460,13 @@
             // Let the current user listener trigger the changes.
           },
           function error(error) {
-            if ("Access denied." == error.reason) {
-              // Access denied is not an error, user hit cancel
-              return;
-            } else {
-              console.error(error);
+            // Access denied is not an error, user hit cancel
+            if ("Access denied." !== error.reason) {
+              this.signinAwares.forEach(function(awareInstance) {
+                awareInstance.fire('google-signin-aware-error', error);
```

Comment:
This seems a bit dodgy, but I don't know how else to get access to the underlying Polymer element.